Polyfills/Window/CMakeLists.txt

```diff
@@ -12,7 +12,9 @@ target_include_directories(Window
 target_link_to_dependencies(Window 
     PUBLIC napi
     PRIVATE base-n
-    PRIVATE JsRuntime)
+    PRIVATE JsRuntime
+    PRIVATE GraphicsInternal)
 
 set_property(TARGET Window PROPERTY FOLDER Polyfills)
 source_group(TREE ${CMAKE_CURRENT_SOURCE_DIR} FILES ${SOURCES})
+ 
```

Comment:
nit: I don't know what is going on here. Wrong line ending maybe?

Reply:
weird, removed.